certified-connectors/OpenExperience/apiDefinition.swagger.json

```diff
@@ -0,0 +1,249 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "Open Experience",
+    "description": "This is an Open Experience connector, which enables you to integrate the Open Experience services with your flows",
+    "version": "1.0",
+    "contact": {
+      "name": "Open Experience Support",
+      "url":"https://openexperience.de/de/hilfe/MS_Datenschnittstelle/MS_Power_Automation_Connector",
+      "email":"support@openexperience.de"
+    }
+  },
+  "host": "openexperience.de",
+  "basePath": "/teamssync",
+  "schemes": [
+    "https"
+  ],
+  "consumes": [
+    "application/json"
+  ],
+  "produces": [
+    "application/json"
+  ],
+  "paths": {
+    "/connector/createProject": {
+      "post": {
+        "responses": {
+          "default": {
+            "description": "default",
+            "schema": {}
+          }
+        },
+        "summary": "Create new project",
+        "operationId": "CreateNewProject",
+        "parameters": [
+          {
+            "name": "Project Settings",
+            "in": "body",
+            "required": false,
+            "schema": {
+              "type": "object",
+              "properties": {
+                "customerId": {
+                  "type": "string",
+                  "description": "customerId",
+                  "title": "Customer",
+                  "x-ms-dynamic-values": {
+                    "operationId": "GetAvailableCustomers",
+                    "value-path": "id",
+                    "value-collection": "accounts",
+                    "value-title": "label"
+                  }
+                },
+                "id": {
+                  "type": "string",
+                  "description": "id",
+                  "title": "Project ID"
+                },
+                "name": {
+                  "type": "string",
+                  "description": "name",
+                  "title": "Project Name"
+                },
+                "address": {
+                  "type": "string",
+                  "description": "address",
+                  "title": "Address"
+                },
+                "contactPhone": {
+                  "type": "integer",
+                  "format": "int32",
+                  "description": "contactPhone",
+                  "title": "Contact Phone"
+                },
+                "contactEmail": {
+                  "type": "string",
+                  "description": "contactEmail",
```

Comment:
@oxTomaszJ All the description values are expected to be free from grammatical and spell errors. Please provide valid description values for all the properties.